packages/site-components/src/VerticalNavigation.tsx

```diff
@@ -17,11 +17,26 @@ export type VerticalNavigationProps = {
 const MenuIcon = ({ open }) => <Icon name={open ? 'chevronDown' : 'chevronRight'} />;
 
 const menuItemStyles = {
-  button: ({ active }) => {
-    return {
-      fontWeight: active ? 'var(--fontWeight-bold)' : 'var(--fontWeight-regular)'
-    };
+  root: {
+    fontSize: '13px'
```

Comment:
needs to be plugged into vanilla extract theme

Reply:
i think Fusion asked for it to be the same size as the content.. i removed  the action1 style we added previously.. if we need it back to 13px, have a look at the lines I removed and we can re-add/refactor as needed